build/native/typography.js

```diff
@@ -1,119 +1,119 @@
-  "sizeLarge": 20,
-  "sizeNormal": 16,
-  "sizeSmall": 14,
-  "sizeXsmall": 12,
-  "sizeMobile3xlarge": 32,
-  "sizeMobile2xlarge": 28,
-  "sizeMobileXlarge": 22,
-  "lineHeightXlarge": 40,
-  "lineHeightLarge": 32,
-  "lineHeightNormal": 24,
-  "lineHeightSmall": 16,
-  "fontFamilySansSerif": "Mulish",
-  "weightSemiBold": 700,
-  "weightBase": 500,
-  "weightLight": 400,
-  "weightStrong": 600,
-  "weightBold": 900,
-  "h1FontFamily": "Mulish",
-  "h1FontWeight": 700,
-  "h1LineHeight": 40,
-  "h1FontSize": 40,
-  "h1LetterSpacing": 0,
-  "h1ParagraphSpacing": 0,
-  "h1TextDecoration": "none",
-  "h1TextCase": "none",
-  "h2FontFamily": "Mulish",
-  "h2FontWeight": 700,
-  "h2LineHeight": 40,
-  "h2FontSize": 32,
-  "h2LetterSpacing": 0,
-  "h2ParagraphSpacing": 0,
-  "h2TextDecoration": "none",
-  "h2TextCase": "none",
-  "h3FontFamily": "Mulish",
-  "h3FontWeight": 700,
-  "h3LineHeight": 32,
-  "h3FontSize": 24,
-  "h3LetterSpacing": 0,
-  "h3ParagraphSpacing": 0,
-  "h3TextDecoration": "none",
-  "h3TextCase": "none",
-  "h4FontFamily": "Mulish",
-  "h4FontWeight": 700,
-  "h4LineHeight": 24,
-  "h4FontSize": 20,
-  "h4LetterSpacing": 0,
-  "h4ParagraphSpacing": 0,
-  "h4TextDecoration": "none",
-  "h4TextCase": "none",
-  "h5FontFamily": "Mulish",
-  "h5FontWeight": 700,
-  "h5LineHeight": 24,
-  "h5FontSize": 16,
-  "h5LetterSpacing": 0,
-  "h5ParagraphSpacing": 0,
-  "h5TextDecoration": "none",
-  "h5TextCase": "none",
-  "bodyFontFamily": "Mulish",
-  "bodyFontWeight": 500,
-  "bodyLineHeight": 24,
-  "bodyFontSize": 16,
-  "bodyLetterSpacing": 0,
-  "bodyParagraphSpacing": 0,
-  "bodyTextDecoration": "none",
-  "bodyTextCase": "none",
-  "footnoteFontFamily": "Mulish",
-  "footnoteFontWeight": 500,
-  "footnoteLineHeight": 16,
-  "footnoteFontSize": 14,
-  "footnoteLetterSpacing": 0,
-  "footnoteParagraphSpacing": 0,
-  "footnoteTextDecoration": "none",
-  "footnoteTextCase": "none",
-  "captionFontFamily": "Mulish",
-  "captionFontWeight": 500,
-  "captionLineHeight": 16,
-  "captionFontSize": 12,
-  "captionLetterSpacing": 0,
-  "captionParagraphSpacing": 0,
-  "captionTextDecoration": "none",
-  "captionTextCase": "none",
-  "mobileH1FontFamily": "Mulish",
-  "mobileH1FontWeight": 700,
-  "mobileH1LineHeight": 40,
-  "mobileH1FontSize": 32,
-  "mobileH1LetterSpacing": 0,
-  "mobileH1ParagraphSpacing": 0,
-  "mobileH1TextDecoration": "none",
-  "mobileH1TextCase": "none",
-  "mobileH2FontFamily": "Mulish",
-  "mobileH2FontWeight": 700,
-  "mobileH2LineHeight": 32,
-  "mobileH2FontSize": 28,
-  "mobileH2LetterSpacing": 0,
-  "mobileH2ParagraphSpacing": 0,
-  "mobileH2TextDecoration": "none",
-  "mobileH2TextCase": "none",
-  "mobileH3FontFamily": "Mulish",
-  "mobileH3FontWeight": 700,
-  "mobileH3LineHeight": 32,
-  "mobileH3FontSize": 22,
-  "mobileH3LetterSpacing": 0,
-  "mobileH3ParagraphSpacing": 0,
-  "mobileH3TextDecoration": "none",
-  "mobileH3TextCase": "none"
-};
\ No newline at end of file
+  letterSpacingBase: 0,
```

Comment:
I guess these quotes are not necessary? Interesting

Reply:
yes i noticed this too locally!